test/integration/framework/master_utils.go

```diff
@@ -141,16 +141,18 @@ func startMasterOrDie(masterConfig *master.Config) (*master.Master, *httptest.Se
 		}
 
 		masterConfig = &master.Config{
-			DatabaseStorage:    etcdStorage,
-			ExpDatabaseStorage: expEtcdStorage,
-			KubeletClient:      client.FakeKubeletClient{},
-			EnableLogsSupport:  false,
-			EnableProfiling:    true,
-			EnableUISupport:    false,
-			APIPrefix:          "/api",
-			ExpAPIPrefix:       "/experimental",
-			Authorizer:         apiserver.NewAlwaysAllowAuthorizer(),
-			AdmissionControl:   admit.NewAlwaysAdmit(),
+			DatabaseStorage:      etcdStorage,
+			ExpDatabaseStorage:   expEtcdStorage,
+			KubeletClient:        client.FakeKubeletClient{},
+			EnableExp:            true,
```

Comment:
Why do we need to add this line? IIRC, we enable exp in test/test-integration.sh through KUBE_API_VERSIONS now.